### ISSUE TITLE
Add constructor for vctAxisAngleRotation3() using axis and angle

### ISSUE THREAD

**Issue by marcinbalicki** [jhu-cisst/cisst#51]
Would be helpful to have an axis and angle constructor:

     vctRodRot3(const vct3 &axis, const double &angle) { ... }

**Comment by adeguet1** [comment]
I'm a bit confused, `vctAxisAngleRotation3` has a constructor from an axis and angle.   Your example is using `vctRodriguezRotation3` which indeed doesn't have an (implicit) constructor from an axis and an angle.   You have to explicitly create a vctAxAnRot3:  

You should be able to do the same with an explicit call to AxAnRot3 constructor:

 `vctRodRot3 rodRot(vctAxAnRot3(vct3(1.0, 0.0, 0.0), cmnPI));`

You should also be able to construct from a vector, i.e.:

 `vctRodRot3 rodRot(vct3(1.0, 0.0, 0.0) * cmnPI);  // this is probably the best answer to your question`

The advantage of the first example is that the constructor will complain if the axis is not normalized:

 `vctRodRot3 rodRot(vctAxAnRot3(vct3(1000.0, 0.0, 0.0), cmnPI));  // this will throw an exception`  

You can enforce that the axis is normalized by the `vctAxAnRot3` constructor using:

 `vctRodRot3 rodRot(vctAxAnRot3(vct3(1000.0, 0.0, 0.0), cmnPI, VCT_NORMALIZE));`

To summarize, I don't think this special case constructor is needed.   Furthermore, to have a consistent API, I would need to add the constructor w/wo the `VCT_NORMALIZE`/`VCT_DO_NOT_NORMALIZE` as well as the `From` methods.

**Comment by marcinbalicki** [comment]
Thanks Anton, 
   Was hoping for a more direct construction but your 3 vector constructor works well and make sense. 
`vctRodRot3 rodRot(vct3(1.0, 0.0, 0.0) * cmnPI); // this is probably the best answer to your question
`
